articles/migration/updating_versions.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Updating Versions
+description: A guide on updating MonoGame when a new release is pushed.
+---
+
+When a new release of MonoGame is pushed, developers will need to manually update their development environment and/or existing project to make use of the new version.  This guide is intended to walk you through the steps of performing these updates.
+
+# Update dotnet Templates
```

Comment:
Dont use H1 here, the H1 is added by the template using the "title" (after all you built that :D)

All subsequent headers should be H2 or H3

Reply:
Heading levels updated